### DISCUSSION TITLE
Trouble getting started

### DISCUSSION THREAD

**Discussion by JonteP** [umair-hassan/best-toolbox#44] [General]
I am afraid this will be a newbie question, but I have no luck playing around with the toolbox.
My first aim was to try something basic, starting a fieldtrip sinewave buffer and see whether I can get the BEST toolbox to read from it. 
After checking that I can actually read from the buffer in Matlab, I first created an input device with the default settings.
I then created a session and added rsEEG Measurement to it and then select the created input device. When clicking compile I get the error: 'Reference to non-existent field 'trials'.
I tried different settings in the protocol with no effect. In fact, regardless of protocol, I get the error 'Reference to non-existent field...' but always pointing to different fields. I managed to figure out that the missing field 'hardware_settings' in one instance was due to my added device having a whitespace in its name.

Something that would help in general would be to rethrow the error to make debugging easier. Otherwise, is there anything obvious I am doing wrong here?

Cheers,
Jonas

**Comment by umair-hassan** [comment]
Hi Jonas, 

Thank you for the reply. Can you please attach the .mat experimental protocol file saved with toolbox. I will reproduce it and then we will be able to debug it quite soon.

**Reply by JonteP** [reply]
Thank you for getting back. I could not get the save button to work (all I get is 'Elapsed time is x seconds.' in the console). I did however save the best_toolbox object in a .mat and attach it here. I am sorry if this is just a bad setup on my part.

[best.zip](https://github.com/umair-hassan/best-toolbox/files/9705113/best.zip)
